frontend/src/components/Overview.vue

```diff
@@ -302,4 +334,7 @@ export default class AttendeesRegistration extends Vue {
 .departmentCount {
   color: rgba(0, 0, 0, 0.6);
 }
+.v-btn.checkin {
+  background-color: #00ff0030;
```

Comment:
Das sieht mir aber nicht nach einem Hex-Code aus.
Das sind 2 Stellen zu viel.

Reply:
This is with alpha channel